.travis.yml

```diff
@@ -135,18 +140,29 @@ script:
     # Insert build information into compiled binary using ldflags
     - CGO_ENABLED=0 go build -ldflags "-X main.Commit=`echo $TRAVIS_COMMIT` -X main.Tag=`echo $TRAVIS_TAG` -X main.Branch=`echo $TRAVIS_BRANCH` -X main.BuildNumber=`echo $TRAVIS_BUILD_NUMBER`"
 
-    # Start test environment
-    - bash ./tests/testenv.sh
+    # Build docker image from docker file
+    - sudo docker build -t $DOCKER_REPOSITORY .
 
-    # Only install acceptance testing dependencies after License checking is complete
-    - npm install validator chance jasmine-node frisby
+    # Clone integration repo. for api testing (purposely after license checking..)
+    - mv integration integration_old && git clone https://github.com/mendersoftware/integration.git
 
-    # Run acceptance tests
-    # Acceptance tests temporarily disabled
-    # - jasmine-node ./tests/images/
 
-    # Build docker image from docker file
-    - sudo docker build -t $DOCKER_REPOSITORY .
+    # Move script to testing directory
+    - mv integration/extra/travis-testing/run-test-environment tests
+
+
+    # Run dependant microservices and the tests service itself
+    - if [[ "$JOB_TYPE" = compile_and_integration_tests* ]]; then
+        NO_CLEANUP=true TESTS_DIR=$PWD/tests ./tests/run-test-environment "integration" $PWD/integration ./tests/docker-compose-integration.yml &
```

Comment:
Didn't look too closely at other integration testing PRs, but this can be rewritten as:

```
NO_CLEANUP=true TESTS_DIR=$PWD/tests \
./tests/run-test-environment \
    "integration" $PWD/integration ./tests/docker-compose-integration.yml &
```

Current formatting does not fit the github's narrow diff view.